dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -153,7 +153,9 @@ export default defineComponent({
         ][]) {
           if (subCategoryObject == null) continue;
           for (const [kpiKey, kpiValue] of Object.entries(subCategoryObject)) {
-            if (kpiValue == null) continue;
+            if (kpiValue == null) {
+              subCategoryObject[kpiKey] = "";
```

Comment:
should be `kpiValue = "";`

Reply:
Can we remove this if block here then? I get your point, that non-null values are preferable. But not here.

Reply:
Yes we can. Removed